src/Library/demos/Status Page/main.css

```diff
@@ -1,6 +1,6 @@
 box {
   background-color: @success_fg_color;
-  box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.3);
+  box-shadow: 2px 2px 5px @warning_fg_color;
```

Comment:
What's the purpose of using these named colors here? Maybe you were thinking of one of the [Miscellaneous](https://gnome.pages.gitlab.gnome.org/libadwaita/doc/1-latest/named-colors.html#miscellaneous-colors) or [Helper Colors](https://gnome.pages.gitlab.gnome.org/libadwaita/doc/1-latest/named-colors.html#helper-colors)?